src/main/scala/io/iohk/ethereum/ledger/Ledger.scala

```diff
@@ -585,19 +588,19 @@ class LedgerImpl(
     def getAccountToPay(address: Address, ws: InMemoryWorldStateProxy): Account = ws.getAccount(address)
       .getOrElse(Account.empty(blockchainConfig.accountStartNonce))
 
-    val minerAddress = Address(block.header.beneficiary)
+    val minerAddress = Address(block.signedHeader.header.beneficiary)
     val minerAccount = getAccountToPay(minerAddress, worldStateProxy)
-    val minerReward = blockRewardCalculator.calcBlockMinerReward(block.header.number, block.body.uncleNodesList.size)
-    val afterMinerReward = worldStateProxy.saveAccount(minerAddress, minerAccount.increaseBalance(UInt256(minerReward)))
-    log.debug(s"Paying block ${block.header.number} reward of $minerReward to miner with account address $minerAddress")
-
-    block.body.uncleNodesList.foldLeft(afterMinerReward) { (ws, ommer) =>
+    val minerReward = blockRewardCalculator.calcBlockMinerReward(block.signedHeader.header.number, block.body.uncleNodesList.size)
```

Comment:
As we have removed the whole usage of ommers, maybe we should temporarely pass 0 here as the ommer length?

Reply:
sure!